tcav/model.py

```diff
@@ -378,3 +378,41 @@ def __init__(self, sess, model_saved_path, labels_path):
                                                   endpoints_v3,
                                                   scope='v3')
     self.model_name = 'InceptionV3_public'
+
+class MobilenetV2Wrapper_public(PublicImageModelWrapper):
+    def __init__(self, sess, model_saved_path, labels_path):
+        self.image_value_range = (-1, 1)
+        image_shape_v2 = [224, 224, 3]
+        endpoints_v2 = dict(
+            input='input:0',
+            prediction='MobilenetV2/Predictions/Reshape:0',
+        )
+
+        self.sess = sess
+        # self.import_prefix = "MobilenetV2"
+
+        super(MobilenetV2Wrapper_public, self).__init__(sess,
+                                                        model_saved_path,
```

Comment:
for the public models, do we provide pointers to labels files? i assume the fetch data/models code gets the saved models necessary

Reply:
The fetch models script will download folders that contain labelmaps. Do you want me to explicitly comment them in the code?

Reply:
just curious. since the download contains it, i think we're good